wremnants/CardTool.py

```diff
@@ -586,7 +592,17 @@ def writeForProcess(self, h, proc, syst, check_systs=True):
             systInfo = self.systematics[syst]
             procDict = self.datagroups.getDatagroups()
             hnom = procDict[proc].hists[self.nominalName]
+            #logger.debug(f"{proc}: {syst}: {h.axes.name}")
+            if systInfo["scalePrefitHistYields"] != None:
+                scaleFactor = systInfo["scalePrefitHistYields"]
+                logger.warning(f"Scaling yields of histogram for syst = {syst} by {scaleFactor}")
+                h = hh.scaleHist(h, scaleFactor, createNew=True)
```

Comment:
For the simple scaling of yields yes.
But for the sum of the nominal I am not totally sure it is easy to do even with a lambda function, because you need to pass a second object that is defined within the code

Reply:
So in any case, for now I didn't change scalePrefitHistYields to make the action done in actionMap. The fact is that the scaling of the histogram is usually not correct, it works here only because the original histogram is already encoding a variation. This feature would be an alternative way to scale the nuisance as opposed to scaling the sigma associated to it in the card, and I'll have to check they do the same thing in the fit (part of the ongoing debugging).